tests/run_all_tests.sh

```diff
@@ -42,7 +43,7 @@ handle_errors () {
 sphinx-build -M doctest docs docs/_build
 
 # Run battery of core FLAX API tests.
-pytest -n 4 tests $PYTEST_OPTS
+pytest -n auto tests $PYTEST_OPTS
```

Comment:
didn't know about `auto`

Reply:
We had it before, It was accidentally removed in the linen refactor